profiler/include/profiler/profile_grouped_conv_bwd_data_impl.hpp

```diff
@@ -125,6 +125,11 @@ bool profile_grouped_conv_bwd_data_impl(int do_verification,
     bool pass = true;
 
     auto run_impl = [&](auto& op_ptr, auto& argument_ptr) {
+        // workspace_sz will be equal to 0 for other layout than NGCHW
+        const std::size_t workspace_sz = op_ptr->GetWorkSpaceSize(argument_ptr.get());
+        DeviceMem workspace_dev(workspace_sz);
```

Comment:
Wouldn't hipMalloc throw inside this function if the requested buffer size will be zero?

Reply:
No, it will handle it